src/main/java/com/google/firebase/auth/FirebaseUserManager.java

```diff
@@ -54,12 +56,27 @@
  */
 class FirebaseUserManager {
 
-  static final String USER_NOT_FOUND_ERROR = "USER_NOT_FOUND_ERROR";
-  static final String USER_CREATE_ERROR = "USER_CREATE_ERROR";
-  static final String USER_UPDATE_ERROR = "USER_UPDATE_ERROR";
-  static final String USER_DELETE_ERROR = "USER_DELETE_ERROR";
-  static final String LIST_USERS_ERROR = "LIST_USERS_ERROR";
-  static final String INTERNAL_ERROR = "INTERNAL_ERROR";
+  static final String USER_NOT_FOUND_ERROR = "user-not-found";
+  static final String INTERNAL_ERROR = "internal-error";
+
+  // Map of server-side error codes to SDK error codes.
+  // SDK error codes defined at: https://firebase.google.com/docs/auth/admin/errors
```

Comment:
I might be missing something, shouldn't this map should be eyed on the error code values in that link e.g. "auth/claims-too-large", not the upper cased version. (later we get from the map based on getErrorCode())

Reply:
The backend server sends the upper-cased error codes. We need to map them to the client-side error codes defined in the public documentation.